erlang.spec

```diff
@@ -319,6 +321,9 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %changelog
+* Tue Oct 4 2022 Marcin Juszkiewicz <marcin.juszkiewicz@linaro.org> - 25.1.2
```

Comment:
Erlang 25.1.2 has not been released yet. There are no upstream source changes. We cannot accept a changelog entry like this.

Reply:
fixed to 25.1.1-2 like it should be